webauthn/resources/webauthn-subframe.sub.html

```diff
@@ -0,0 +1,36 @@
+<!DOCTYPE html>
+<meta charset="utf-8">
+<title>WebAuthn test subframe</title>
+<script src=../helpers.js></script>
+<script src="/resources/testdriver.js"></script>
+<script src="/resources/testdriver-vendor.js"></script>
+
+<script>
+    "use strict";
+
+    function reportResult(outcome, errorMessage) {
+        window.top.postMessage({"type": "result", "result": outcome, "errorMessage": errorMessage}, "*");
+    }
+
+    async function createCredentialAndReportResult(addUserActivation) {
+        if (addUserActivation) {
+            await test_driver.bless("create credential");
+        }
+
+        try {
+            await createCredential();
+            reportResult("success");
+        } catch (e) {
+            reportResult("failure", e.message);
+        }
+    }
+
+    window.addEventListener("message", e => {
+        if (e.data.type == "create-credential") {
+            createCredentialAndReportResult(e.data.addUserActivation);
+        }
```

Comment:
We should throw here if we get an unrecognized message.

Reply:
Done, though it won't be directly visible in tests sadly - one has to check the console and/or devtools logs.

Reply:
- [ ] > - - [ ] # - [ ] @~~@~~@ ~~ ~~